File: csharp/gabaritos/gabarito3.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Digite um número: ");
        int num = int.Parse(Console.ReadLine());
        if (num % 2 == 0)
        {
            Console.WriteLine("Par");
        }
        else
        {
            Console.WriteLine("Ímpar");
        }
    }
}

File: cpp/gabaritos/gabarito3.cpp
#include <iostream>

int main() {
    int num;
    std::cout << "Digite um número: ";
    std::cin >> num;
    if (num % 2 == 0) {
        std::cout << "Par" << std::endl;
    } else {
        std::cout << "Ímpar" << std::endl;
    }
    return 0;
}